sdk/ml/azure-ai-ml/tests/test_configs/pipeline_jobs/helloworld_pipeline_job_register_pipeline_and_node_output.yaml

```diff
@@ -0,0 +1,21 @@
+display_name: register_both_output
+type: pipeline
+jobs:
+  parallel_body:
+    type: command
+    inputs:
+      component_in_path:
+        type: uri_file
+        path: https://dprepdata.blob.core.windows.net/demo/Titanic.csv
+    component: ../components/helloworld_component.yml
+    outputs:
+      component_out_path:
+        name: node_output
+        version: '1'
+outputs:
+  pipeline_out_path:
```

Comment:
Add a cli test when node output bind to pipeline output

Reply:
added in test_register_output_yaml